ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -228,8 +228,9 @@ joinWindow
     ;
 
 withinExpression
-    : '(' joinWindowSize ',' joinWindowSize ')' # joinWindowWithBeforeAndAfter
-    | joinWindowSize # singleJoinWindow
+    : '(' joinWindowSize ',' joinWindowSize ')'         # joinWindowWithBeforeAndAfter
```

Comment:
Do we need a 4th line for `before/after with grace` ?

Reply:
I'll add it in a follow-up PR (part of parallelized work).